client/src/App.js

```diff
@@ -22,7 +23,8 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
```

Comment:
I believe what you want to do is maybe send the `addToSavedList` method to another component to be invoked. 

`addToSavedList` takes in an argument (the target movie) and then adds it to the array of movies. 

The way this would look is:
```suggestion
        <SavedList addToSavedList={this.addToSavedList} list={this.state.savedList} />
```
Then in the SavedList component you would call it with `props.addToSavedList` or `this.props.addToSavedList` depending on whether it's a functional or class component.